From acf34999930dcbf3244d506fc131f8d6f8be72ae Mon Sep 17 00:00:00 2001
From: Maryna Malakhova <maryna.malakhova@globallogic.com>
Date: Wed, 14 Apr 2021 12:00:16 +0300
Subject: [PATCH] 11_RTC_DRIVER: Modify am335x-boneblack.dts

Change rtc record in the am335x-boneblack.dts
Use "maxim,ds3231x" as compatible string

Signed-off-by: Maryna Malakhova <maryna.malakhova@globallogic.com>
---
 arch/arm/boot/dts/am335x-boneblack.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-boneblack.dts b/arch/arm/boot/dts/am335x-boneblack.dts
index d154d3133c16..4e15a2aa862b 100644
--- a/arch/arm/boot/dts/am335x-boneblack.dts
+++ b/arch/arm/boot/dts/am335x-boneblack.dts
@@ -26,3 +26,11 @@
		opp-supported-hw = <0x06 0x0100>;
	};
 };
+
+&i2c2 {
+	ds3231: ds3231@68 {
+		compatible = "maxim,ds3231x";
+		reg = <0x68>;
+	};
+};
+
--
2.25.1

